### PR TITLE
*: update kvproto.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut  @shenli @coocood  @disksing

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go [hunk 1]
```diff
@@ -373,7 +373,15 @@ func rpcConnect(addr string) (net.Conn, error) {
 	}
 
 	for _, url := range urls {
-		conn, err := net.DialTimeout("tcp", url.Host, connectPDTimeout)
+		var conn net.Conn
+		switch url.Scheme {
+		// used in tests
+		case "unix", "unixs":
```
Comment (shenli) [inline _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go]: PD supports unix scoket?

Comment (siddontang) [inline _vendor/vendor/github.com/pingcap/pd/pd-client/rpc_worker.go]: yes, but now only for test. but if you want to use in local, it is ok.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**